[feat]: brand new enums (#69)
soramitsu/scale-codec-js-library: packages/definition-compiler/CHANGELOG.md

# @scale-codec/definition-compiler

## 3.0.0

### Minor Changes

> - 7e81bbc: **refactor**: generate imports/exports each on a new line

Reviewer: ```suggestion
- 7e81bbc: **refactor**: generate each import/export on a new line
```

> - 7e81bbc: **refactor**: update generation of enums due to the major update of their design

Reviewer: ```suggestion
- 7e81bbc: **refactor**: update enum generation following the major update in enum design
```
